tests/migrations/0001_initial.py

```diff
@@ -0,0 +1,155 @@
+# Generated by Django 2.2.19 on 2021-03-31 07:46
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```

Comment:
Django 2.2+ runs syncdb (for apps without migrations) before migrations. This means that the tests failed (as the models are dependent on `django.contrib.contenttypes` tables), which is why I added initial migrations for the app.